usaspending_api/conftest_helpers.py

```diff
@@ -60,28 +64,52 @@ def _add_contents(self, **options):
         The view is only needed to load the transactions into Elasticsearch so it is dropped after each use.
         """
-        view_sql_file = "award_delta_view.sql" if self.index_type == "awards" else "transaction_delta_view.sql"
+        if self.index_type == "awards":
+            view_sql_file = f"{settings.ES_AWARDS_ETL_VIEW_NAME}.sql"
+            view_name = settings.ES_AWARDS_ETL_VIEW_NAME
+            es_id = "award_id"
+        elif self.index_type == "covid19_faba_":
+            view_sql_file = f"{settings.ES_COVID19_FABA_ETL_VIEW_NAME}.sql"
+            view_name = settings.ES_COVID19_FABA_ETL_VIEW_NAME
+        else:
+            view_sql_file = f"{settings.ES_TRANSACTIONS_ETL_VIEW_NAME}.sql"
+            view_name = settings.ES_TRANSACTIONS_ETL_VIEW_NAME
+            es_id = "transaction_id"
+
         view_sql = open(str(settings.APP_DIR / "database_scripts" / "etl" / view_sql_file), "r").read()
         with connection.cursor() as cursor:
             cursor.execute(view_sql)
-            if self.index_type == "transactions":
-                view_name = settings.ES_TRANSACTIONS_ETL_VIEW_NAME
-            else:
-                view_name = settings.ES_AWARDS_ETL_VIEW_NAME
             cursor.execute(f"SELECT * FROM {view_name};")
-            transactions = ordered_dictionary_fetcher(cursor)
+            records = ordered_dictionary_fetcher(cursor)
             cursor.execute(f"DROP VIEW {view_name};")
-
-        for transaction in transactions:
+            if self.index_type == "covid19_faba_":
+                records = transform_covid19_faba_data(
+                    TaskSpec(
+                        name="worker",
+                        index=self.index_name,
+                        sql=view_sql_file,
+                        view=view_name,
+                        base_table="financial_accounts_by_awards",
+                        base_table_id="financial_accounts_by_awards_id",
+                        field_for_es_id="financial_account_distinct_award_key",
+                        primary_key="award_id",
+                        partition_number=1,
+                        is_incremental=False,
+                    ),
+                    records,
+                )
+        for record in records:
             # Special cases where we convert array of JSON to an array of strings to avoid nested types
             routing_key = options.get("routing", settings.ES_ROUTING_FIELD)
-            routing_value = transaction.get(routing_key)
+            routing_value = record.get(routing_key)
             if self.index_type == "transactions":
-                transaction["federal_accounts"] = self.convert_json_arrays_to_list(transaction["federal_accounts"])
+                record["federal_accounts"] = self.convert_json_arrays_to_list(record["federal_accounts"])
+            elif self.index_type == "covid19_faba_":
+                es_id = record.pop("_id")
```

Comment:
This was some odd behavior with the `transform_covid19_faba_data` - it manually sets the `_id` field, but the way that we index the records in this test setup will throw an error if the `_id` field is set, so we pop it to prevent that error.